src/cms/cms.js

```diff
@@ -1,10 +1,28 @@
 window.CMS_MANUAL_INIT = true;
 
+import React from 'react';
 import CMS from 'decap-cms-app';
 import PostPreview from './previewTemplate';
 import styles from '!css-loader!../style.css';
 import configJson from './config.js';
 
+const tipsControl = () => {
+  return (
+    <div>
+      <a
+        target="_blank"
+        href="https://github.com/SSWConsulting/SSW.Rules.Content/wiki/Netlify-CMS-%F0%9F%90%9B----Failed-to-save"
+        rel="noreferrer"
+        style={{ textDecoration: 'underline' }}
```

Comment:
@KristenHu we typically use tailwind styling on rules... any reason we can't use it here?

Reply:
> @KristenHu we typically use tailwind styling on rules... any reason we can't use it here?

Tailwindcss doesn't work on custom widgets, that's why I use style attribute instead, I doubt it's more of a package issue.